toml.md

```diff
@@ -128,12 +128,13 @@ to use bare keys except when absolutely necessary.
 ```
 
 A bare key must be non-empty, but an empty quoted key is allowed (though
-discouraged).
+discouraged). You cannot use multi-line strings in a quoted key.
```

Comment:
I'd change this to one of: "You cannot use multi-line strings to make/write a quoted key."

Reply:
Yeah, "in a quoted key" is perhaps not the best. After going back and forth on a few alternatives I ended up with:

> You cannot use multi-line strings to define quoted keys.

Whatdayathink?

Reply:
Has my vote 👍🏻